### PR TITLE
Use `Inter` font as the sans font

### PR DESCRIPTION
**Pull request by rithviknishad**
## Proposed Changes

- Use Inter as sans font as per [figma design](https://www.figma.com/file/fqDMgeN2m4TCSz5KobZaKW/CoronaSafe-Network-Care?node-id=1148%3A2)
- Fix overweights due to the new font.

Sample screenshots of a few pages:

image

image

image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Review by Ashesh3 — APPROVED** [review]
The font looks professional!

**Review by cp-Coder — APPROVED** [review]

**Comment by nihal467** [comment]
@rithviknishad 
image

- [ ] coronasafe network used to be in one line, are now in two lines, is it intentional change

**Comment by nihal467** [comment]
@aparnacoronasafe can you check the entire platform and let me know your opinion, about the font change

**Comment by rithviknishad** [comment]
@nihal467 I've fixed the login page title issue.

File: tailwind.config.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ module.exports = {
   important: true,
   theme: {
     extend: {
+      fontFamily: {
+        sans: "Inter",
```
Comment (bodhish) [inline tailwind.config.js]:
```suggestion
        sans: "Inter", sans-serif;
```

File: src/style/index.css [hunk 2]
```diff
@@ -10,7 +10,7 @@ html {
 }
 
 body {
-  font-family: "Open Sans", sans-serif;
+  font-family: 'Inter', 'Open Sans', sans-serif;
```
Comment (bodhish) [inline src/style/index.css]:
```suggestion
```
This won't be required as we are setting `font-family` to `font-sans` in line 15

File: public/index.html [hunk 3]
```diff
@@ -27,7 +27,9 @@
     type="image/x-icon" />
   <link rel="icon" href="https://cdn.coronasafe.network/care-manifest/images/icons/favicon.ico" type="image/x-icon" />
   <link rel="manifest" href="/manifest.webmanifest" crossorigin="use-credentials" />
-  <link rel="preconnect" href="https://fonts.gstatic.com" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```
Comment (bodhish) [inline public/index.html:31]:
```suggestion
```
Why do we need this 🤔

**Review by bodhish — APPROVED** [review]
LGTM

File: tailwind.config.js (hunk 1 above)
Comment (bodhish) [reply tailwind.config.js]: Lets add a fallback

File: public/index.html (hunk 3 above)
Comment (rithviknishad) [reply public/index.html:31]: @bodhish isn't it a standard practice to preconnect to google fonts for faster loads?

https://www.cdnplanet.com/blog/faster-google-webfonts-preconnect/

Should I still remove it?

**Comment by nihal467** [comment]
@rithviknishad once the review and work is completed, label it to need testing

File: public/index.html (hunk 3 above)
Comment (rithviknishad) [reply public/index.html:31]: https://share.firefox.dev/3T7PCFv

image

**Comment by nihal467** [comment]
@aparnacoronasafe it looks good to me, can you give it a check

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**